Utilities/IntervalFiles/README.md

```diff
@@ -0,0 +1,237 @@
+* Bucket Mirror: `gs://broad-dsde-methods-hydro-gen-truth-data-public/NIST/IntervalFiles/GRCh38_gc15_slop50.bed.gz`
+* Original File: [here](https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/release/genome-stratifications/v3.1/GRCh38/GCcontent/GRCh38_gc15_slop50.bed.gz)
+* Original Documentation: [here](https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/release/genome-stratifications/v3.1/GRCh38/GCcontent/GRCh38-GCcontent-README.md)
+* Percent of Genome: 0.2%
+* Interesting Overlaps:
+  * % of LowGC in LowComplexityRegion: 60% (Jaccard = 4.5%)
+
+#### LowMappability
+* Summary: Contains regions in reference which are homologous to other regions of length <= 250 exact matches
+  (high stringency), or <= 100 with up to 2 mismatch/1 indel (low stringency). Refinements to either subset is available
+  from NIST.
+* Bucket Mirror: `gs://broad-dsde-methods-hydro-gen-truth-data-public/NIST/IntervalFiles/GRCh38_lowmappabilityall.bed.gz`
+* Original File: [here](https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/release/genome-stratifications/v3.1/GRCh38/mappability/GRCh38_lowmappabilityall.bed.gz)
+* Original Documentation: [here](https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/release/genome-stratifications/v3.1/GRCh38/mappability/GRCh38-mappability-README.md)
+* Percent of Genome: 8%
+* Interesting Overlaps:
+    * % of LowMappability in Homopolymers: 2% (Jaccard = 1.5%)
+    * % of LowMappability in LowComplexityRegion: 1.5% (Jaccard = 1%)
+    * % of LowMappability in SegDup: 44% (Jaccard = 36%)
+    * % of LowMappability in AllDifficult: 100% (contained; Jaccard = 39%)
+
+#### Homopolymers
+* Summary: Contains regions in reference which are homopolymers (repeated base) of length >= 6 and imperfect homopolymers
+  (homopolymers interrupted by 1 different base) of length >= 10, with 5 bases added on both sides for padding.
+* Bucket Mirror: `gs://broad-dsde-methods-hydro-gen-truth-data-public/NIST/IntervalFiles/GRCh38_AllHomopolymers_gt6bp_imperfectgt10bp_slop5.bed.gz`
+* Original File: [here](https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/release/genome-stratifications/v3.1/GRCh38/LowComplexity/GRCh38_AllHomopolymers_gt6bp_imperfectgt10bp_slop5.bed.gz)
+* Original Documentation: [here](https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/release/genome-stratifications/v3.1/GRCh38/LowComplexity/GRCh38-LowComplexity-README.md)
+* Percent of Genome: 2.7%
+* Interesting Overlaps:
+  * % of Homopolymers in LowMappability: 6% (Jaccard = 1.5%)
+  * % of Homopolymers in LowComplexityRegion: 37% (Jaccard = 26%)
+  * % of Homopolymers in SegDup: 5.6% (Jaccard = 2%)
+  * % of Homopolymers in AllDifficult: 100% (contained; Jaccard = 13%)
+
+#### LowComplexityRegion
+* Summary: Contains regions of "low complexity", using the symmetric DUST algorithm. 
+* Bucket Mirror: `gs://broad-dsde-methods-hydro-gen-truth-data-public/NIST/IntervalFiles/LCRFromHengHg38.bed`
```

Comment:
This isn't really from NIST, is it ok that you're putting it in a folder with NIST in the name?

Reply:
Good point, I relocated them to a better bucket path without NIST in it.